packages/ingest/src/sqs.ts

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+import Logger from '@cumulus/logger';
+import { SQSMessage } from '@cumulus/aws-client/SQS';
+const { envUtils } = require('@cumulus/common');
+const { s3PutObject } = require('@cumulus/aws-client/S3');
+const { deleteS3Object } = require('@cumulus/aws-client/S3');
+const logger = new Logger({ sender: '@cumulus/ingest/sqs' });
+
+function getS3KeyForArchivedMessages(stackName: string, messageId?: string) {
```

Comment:
It's optional because the Message interface has `MessageId` as optional. Though, it seems like all messages should be guaranteed a message ID. Will update.